.github/workflows/lint-and-test.yml

```diff
@@ -116,7 +116,7 @@ jobs:
           SA_REPLAY_SKIP_BINARY_DOWNLOAD: 1
 
       - name: Run tests
-        run: yarn jest --testTimeout=15000 --runInBand
+        run: yarn jest --testTimeout=60000 --runInBand
```

Comment:
That's a major increase.

Reply:
It was an attempt to workaround these sporadically failing tests that happens almost exclusively on windows. I didn't spend the time to figure out how to change for only windows.